pkg/reconciler/taskrun/taskrun.go

```diff
@@ -744,18 +744,6 @@ func (c *Reconciler) createPod(ctx context.Context, ts *v1beta1.TaskSpec, tr *v1
 		return nil, err
 	}
 
-	ts, err = resources.AddInputResource(ctx, c.KubeClientSet, c.Images, rtr.TaskName, ts, tr, inputResources)
-	if err != nil {
-		logger.Errorf("Failed to create a pod for taskrun: %s due to input resource error %v", tr.Name, err)
-		return nil, err
-	}
-
```

Comment:
why is this removed? I don't think it should be

Reply:
Thanks Lee. Restored as we still have image resources to be removed but can be an input.